scripts/compile/compile-stage.ps1

```diff
@@ -48,15 +48,6 @@ if (!$?) {
     Exit 1
 }
 
-# Build the projects that we are going to ship as nuget packages
-$ProjectsToPack | ForEach-Object {
```

Comment:
Don't do this. Let's not rely on the side-effect that the projects we want to pack are being built by the projects we are trying to publish to guarantee that they are there.

Let's be explicit about building the projects to pack. Also, if you change the to package projects output, we need to update the VSO definitions that are picking up on the packages.